com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -57,19 +57,32 @@ public static ActionSpec MakeContinuous(int numActions)
         /// <param name="branchSizes">The array of branch sizes for the discrete action space.  Each index
         /// contains the number of actions available for that branch.</param>
         /// <returns>An Discrete ActionSpec initialized with the array of branch sizes.</returns>
-        public static ActionSpec MakeDiscrete(int[] branchSizes)
+        public static ActionSpec MakeDiscrete(params int[] branchSizes)
```

Comment:
Just a bit syntactic sugar so you can say `MakeDiscrete(1, 2, 3)` instead of `MakeDiscrete(new[] {1, 2, 3})`. I can revert if you're not a fan.